### PR TITLE
button disabled if no EC2 resources

### PR DESCRIPTION
**Pull request by ariellegordon**
- [X] Button is now disabled if no EC2 resources are available

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ node_modules
 /styleguide
 .coverage
 .DS_Store
+.prettierrc.json
```
Comment (helfi92) [inline .gitignore]: Taskcluster web uses https://github.com/mozilla-frontend-infra/neutrino-preset-mozilla-frontend-infra and this includes prettier. We can safely remove `.prettierrc.json` from here.

**Comment by ariellegordon** [comment]
Hi @helfi92 - I made the requested changes on the same branch. Let me know if anything else needs to be changed. Thanks!

File: .gitignore (hunk 1 above)
Comment (ariellegordon) [reply .gitignore]: Hi @helfi92. Did you mean to also remove .prettierrc.json from my git repo? Thanks.

**Comment by ariellegordon** [comment]
@helfi92 Hi - I removed the .prettierrc.json from the PR. Thanks!

**Comment by helfi92** [comment]
Let me know if you need help :)

**Comment by ariellegordon** [comment]
@helfi92 Hi! My fork of the Taskcluster repo appears to not include the newer commits to the repo, even when I pull down from master. Is it possible I could manually update the yarn.lock and push those changes to my local branch?
screen shot 2018-10-10 at 4 52 37 pm

**Comment by helfi92** [comment]
The newer commits are on https://github.com/taskcluster/taskcluster-web.git. The origin remote is pointing to your fork. You can add another remote `upstream` with:

```bash
git remote add upstream  https://github.com/taskcluster/taskcluster-web.git
```

Once that's done, you can do `git pull upstream master --rebase` to rebase on master.

**Comment by ariellegordon** [comment]
Thanks @helfi92. I have performed those commands and I think everything should be up to date now. Let me know if there are any issues.

**Comment by helfi92** [comment]
The changes are still there: https://github.com/taskcluster/taskcluster-web/pull/151/files

Once the rebase is done, you can push your changes:

‘’’bash
git push origin working-branch —force
‘’’

PS., the rebase should be done on the working-branch

**Comment by ariellegordon** [comment]
Hi @helfi92, let me know if everything worked this time. Thank you for your help!

**Comment by helfi92** [comment]
Seems to be good :) I'll test the code tomorrow before merging.

**Pull request merged**